packages/mongodb-memory-server-core/src/util/getport/index.ts

```diff
@@ -48,10 +49,10 @@ export async function getFreePort(
   firstPort?: number,
   max_tries: number = MAX_DEFAULT_TRIES
 ): Promise<number> {
-  // use "Date" as a semi-random value to lessen conflicts between simultaneous tests
-  firstPort = firstPort || validPort(Date.now());
+  // Get a random value from crypto to use as first port if none is given
+  firstPort = firstPort || validPort(crypto.randomInt(MIN_PORT, MAX_PORT));
```

Comment:
```suggestion
  firstPort = firstPort || validPort(crypto.randomInt(MIN_PORT, MAX_PORT + 1));
```

the documentation says that parameter `max` is exclusive, but `MAX_PORT` is inclusive (a number that should be a valid port)